### PR TITLE
SBS-4035: add more integration tests for the geospatial api

### PR DESCRIPTION
**Pull request by vincent-cognite**

### CODE REVIEW

**Review by TerjeWiigMathisen — COMMENTED** [review]
This looks very interesting, but I'm afraid I don't understand it well enough to approve.

File: tests/tests_integration/test_api/test_geospatial.py [hunk 1]
```diff
@@ -11,26 +12,52 @@
...
+
+
+@pytest.fixture()
+def test_feature_type(cognite_domain):
+    COGNITE_CLIENT.geospatial.set_current_cognite_domain(cognite_domain)
     external_id = f"FT_{uuid.uuid4().hex[:10]}"
     feature_type = COGNITE_CLIENT.geospatial.create_feature_types(
         FeatureType(
-            external_id=external_id, attributes={"temperature": {"type": "DOUBLE"}}, cognite_domain=request.param
+            external_id=external_id, attributes={"temperature": {"type": "DOUBLE"}}, cognite_domain=cognite_domain
```
Comment (tuanng-cognite) [inline tests/tests_integration/test_api/test_geospatial.py:25]: I did not notice the attribute `cognite_domain` in the constructor. Retrieve it from `COGNITE_CLIENT.geospatial.set_current_cognite_domain(cognite_domain)` is not enough, right?

**Review by tuanng-cognite — APPROVED** [review]
Only small comment

File: tests/tests_integration/test_api/test_geospatial.py (hunk 1 above)
Comment (vincent-cognite) [reply tests/tests_integration/test_api/test_geospatial.py:25]: You could in theory be working with feature types from 2 domains at the same time. Unclear if that is required at this point.

**Pull request merged**